src/pubsub/mod_pubsub.erl

```diff
@@ -349,6 +365,10 @@ store_config_in_ets(Host, ServerHost, Opts, Plugins, NodeTree, PepMapping) ->
                                      fun(A) when is_list(A) -> filter_node_options(A) end, []),
     ItemPublisher = gen_mod:get_opt(item_publisher, Opts,
                                     fun(A) when is_boolean(A) -> A end, false),
+    ProxyBackend = gen_mod:get_opt(proxy_backend, Opts, fun(A) when is_atom(A) -> A end,
+                                   mod_pubsub_db_backend),
+    ProxyCache = gen_mod:get_opt(proxy_cache, Opts, fun(A) when is_atom(A) -> A end,
```

Comment:
seems that you are introducing new options to the pubsub module, can you document them?